js/main.js

```diff
@@ -142,18 +142,10 @@ document.addEventListener('DOMContentLoaded', function () {
 });
 
 document.addEventListener('click', function (event) {
-  if (event.target.tagName !== 'A') {
```

Comment:
Oops! you should have kept this conditional, this is our check to see if the user actually clicked an anchor tag 👍